src/Traces.Web/ViewModels/BaseViewModel.cs

```diff
@@ -27,17 +25,14 @@ protected BaseViewModel(IHttpContextAccessor httpContextAccessor, IRequestContex
         protected async Task<bool> InitializeContextAsync()
```

Comment:
basically you can change this to

```suggestion
        protected async Task InitializeContextAsync()
```